screens/auth/SignUpScreen.js

```diff
@@ -289,16 +383,16 @@ function validate(fieldName, value) {
 
 // For handling errors within the form. This is strecth to handle the
 // onBlur thing that I talked about.
-const TextField = props => (
-  <View>
-    <TextInput
-      style={props.style}
-      placeholder={props.placeholder}
-      secureTextEntry={props.secureTextEntry}
-    />
-    <TextInput value={props.error ? <Text>{props.error}</Text> : null} />
-  </View>
-);
+// const TextField = props => (
```

Comment:
Can we get rid of this please? (I realized that you literally implemented the 'onBlur' thing being referenced here LOL)